templates/remove-unused-istags.sh.j2

```diff
@@ -12,7 +12,7 @@ autorebuild_imagestream_tags=$(
 delete_threshold=$(date --date="7 days ago" --iso-8601=seconds --utc)
 
 old_imagestream_tags=$(
-    $OC get imagestreamtags -o json | \
+    $OC get imagestreamtags --chunk-size=0 -o json | \
```

Comment:
Could add a comment here to explain why this is needed?

Reply:
added